benchmark/run_phylomizer.sh

```diff
@@ -0,0 +1,122 @@
+# Copyright (c) 2015--, The WGS-HGT Development Team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+# usage: run Phylomizer to build phylogenetic trees of individual gene families
+# notes: the official "KMM" protocol is used
+
+set -eu
+source $(dirname "$0")/utils.sh
+args=(
+    gene_fa_dir  # protein sequences in multi-fasta format (*.fa), one file per gene family
+    gene_tree_dir  # output gene trees will be saved to this directory
+    output_fp
+    scripts_dir
+    phylomizer_install_dir
+    phylomizer_config_fp
+    threads
+    stdout
+    stderr
+    verbose
+)
+get_args "$@"
+
+$verbose && echo "Running Phylomizer .."
+
+# set up
+PWD=$(pwd)
+mkdir -p ${working_dir}/phylomizer
+if [ "${phylomizer_config_fp}" == "None" ]
+then
+    phylomizer_config_fp=${working_dir}/phylomizer/config.txt
+    # modified KMM protocol
+    # minimum number of sequences: 10
+    # only maximum likelihood, but not neighbor joining, is to be conducted
+    echo "
+    verbose             parameter    1
+    residue_datatype    parameter    protein
+    force_seed_sequence parameter	   True
+    alignment           mode         kalign muscle mafft
+    consensus           mode         m_coffee
+    trimming            mode         trimal
+    both_direction      parameter	   True
+    min_seqs            parameter	   10
+    in_letter           parameter    U:B
+    in_letter           parameter    O:Z
+    muscle              binary
+    muscle_params       parameter
+    mafft               binary
+    mafft_params        parameter    --auto
+    kalign              binary
+    kalign_params       parameter    -f fasta
+    m_coffee            binary       t_coffee
+    m_coffee_params     parameter    -n_core 1 -output fasta -quiet
+    trimal              binary
+    trimal_params       parameter    -phylip -gt 0.1
+    trimal_compare      parameter    -ct 0.1667
+    readal              binary
+    tree                mode         phyml
+    evol_models         parameter    JTT WAG MtREV VT LG Blosum62 Dayhoff
+    numb_models         parameter    2
+    tree_approach       mode         ml
+    ml                  parameter    -b -2 -o tlr
+    phyml               binary
+    phyml_params        parameter	   -d aa -f e -v e -a e -c 4 --no_memory_check
+    " > ${phylomizer_config_fp}
+fi
+$verbose && echo "Configuration file:"$'\n'"  ${phylomizer_config_fp}"
+
+# command
+cmd="python ${phylomizer_install_dir}/source/phylomizer.py -i input.fa --steps alignments trees -c ${phylomizer_config_fp} -o ."
+$verbose && echo "Command:"$'\n'"  $cmd"
+
+TIMEFORMAT='%U %R'
+total_user_time="0.0"
+total_wall_time="0.0"
+
+printf "#Phylomizer\n" >> $output_fp
+
+i=0
+cd ${working_dir}/phylomizer
+for gene_fa in $gene_fa_dir/*.fa
```

Comment:
Will this not give the full dir path ($gene_fa_dir and up) as $gene_fa? Don't you want to do some manipulation to actually just get the gene and not the path?

```
>>> for bar in $foodir/*.tsv
>>> do
>>>     echo $bar
>>>     echo ${bar%.tsv}
>>>     g=${bar##*/}
>>>     echo ${g%.tsv}
>>> done

./quast/report.tsv
./quast/report
report
./quast/transposed_report.tsv
./quast/transposed_report
transposed_report

```